docs/content/Getting-Started/user-quickstart-test.md

```diff
@@ -164,16 +164,21 @@ mkdir -p ${PWD}/temp-space-config
            end="<!--kubestellar-apply-syncer-end-->"
          %}
 
-Wait for the mailbox controller to create the corresponding mailbox workspaces and remember them.
+Wait for the mailbox controller to create the corresponding mailbox spaces and remember them.
 
 ```shell
-# TODO: the mailbox workspaces have not been converted into spaces yet.
-KUBECONFIG=ks-core.kubeconfig kubectl ws root
-while [ $(KUBECONFIG=ks-core.kubeconfig kubectl get workspaces | grep -c -e -mb-) -lt 2 ]; do sleep 10; done
-MB1=$(KUBECONFIG=ks-core.kubeconfig kubectl get workspaces -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster1") | .name')
+while [ $(KUBECONFIG=$SM_CONFIG kubectl get spaces -A | grep -c -e -mb-) -lt 2 ]; do sleep 10; done
+MB1=$(KUBECONFIG=$SM_CONFIG kubectl get spaces -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster1") | .name')
 echo The mailbox for ks-edge-cluster1 is $MB1
-MB2=$(KUBECONFIG=ks-core.kubeconfig kubectl get workspaces -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster2") | .name')
+MB2=$(KUBECONFIG=$SM_CONFIG kubectl get spaces -n spaceprovider-default -o json | jq -r '.items | .[] | .metadata | select(.annotations ["edge.kubestellar.io/sync-target-name"] == "ks-edge-cluster2") | .name')
 echo The mailbox for ks-edge-cluster2 is $MB2
+
+MB1_SPACE="${PWD}/temp-space-config/${MB1}"
```

Comment:
(1) I would put something in this variable name indicating that it holds the pathname of a kubeconfig file.

(2) This is about the "mailbox space" for ks-edge-cluster1, right? I would not reorder to put the "1" between "mailbox" and "space". I would name this variable something like "MBS1_KUBECONFIG".

(3) I would make this filename end with ".kubeconfig".